content/guides/admin-set-up/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Set up your company for success with Docker
+linkTitle: Admin set up 
+summary: Get the most out of Docker by streamlining workflows, standardizing development environments, and ensuring smooth deployments across your company.
+description: Learn how to onboard your company and take advantage of all of the Docker products and features. 
+levels: [intermediate]
+params:
+  featured: true
```

Comment:
Should this be a featured guide?

Reply:
Yes. It should be a featured guide. The Docker Compose guide has most views compared to DBC and Scout. So, happy to move one of those to the guides collection.